test/e2e/specs/fakerp/admin.go

```diff
@@ -57,9 +56,8 @@ var _ = Describe("Openshift on Azure admin e2e tests [Fake]", func() {
 
 	It("should ensure no unnecessary VM rotations occured", func() {
 		Expect(os.Getenv("RESOURCEGROUP")).ToNot(BeEmpty())
-		azurecli, err := azure.NewClientFromEnvironment(true)
-		Expect(err).ToNot(HaveOccurred())
-
+		azurecli := sanity.Checker.FakeRPClient
+		Expect(azurecli).NotTo(BeNil())
```

Comment:
unnecessary (throughout)

Reply:
Refactored the tests to not use the "azurecli" or "cli" variable.
Yes, forgot that client creation doesn't perform a dial, so it won't be nil.